File: ling-programacao/c/lista-1/ex04.c
#include <stdio.h>

int main() {
    float temperatura_fahrenheit, temperatura_celsius;

    printf("Digite a temperatura em Fahrenheit: ");
    scanf("%f", &temperatura_fahrenheit);

    temperatura_celsius = (temperatura_fahrenheit - 32) * 5 / 9;

    printf("%.2f Fahrenheit equivale a %.2f Celsius.\n", temperatura_fahrenheit, temperatura_celsius);

    return 0;
}

File: ling-programacao/go/lista-1/ex04.go
package main

import "fmt"

func Conversao(fahrenheit float64) float64 {
    celsius := (fahrenheit - 32) * 5 / 9
    return celsius
}

func main() {
    var fahrenheit float64
    fmt.Print("Digite a temperatura em Fahrenheit: ")
    fmt.Scan(&fahrenheit)

    celsius := Conversao(fahrenheit)
    fmt.Printf("A temperatura em Celsius é: %.2f\n", celsius)
}